### DISCUSSION TITLE
Be able to remove the new forum style

### DISCUSSION THREAD

**Discussion by VFDan** [ScratchAddons/ScratchAddons#790] [Ideas] [answered]
The new forum style is forced. 
![image](https://user-images.githubusercontent.com/44413950/99823787-ce013480-2b22-11eb-9aa5-66f96e26867b.png)
I could not find a way to turn it off via the extension, I have to turn the whole extension off via Firefox.
The site-wide dark theme that I use (and develop) does not work with this style, and I preferred the old style.

**Comment by TheColaber** [comment, marked as answer]
You have 2.0 -> 3.0 addon enabled. It makes the forum more 3.0ish

**Reply by WorldLanguages** [reply]
Yeah, nothing is forced on Scratch Addons. Having all addons disabled should give you the same result as turning off the extension.

**Comment by VFDan** [comment]
Oh. However, that change should be listed in the extension description.

**Reply by TheColaber** [reply]
Maybe a setting? For forums...

**Reply by WorldLanguages** [reply]
Hmm, the description of the addon clearly says `Makes Scratch 2.0-styled pages look like Scratch 3.0`, the forums are still 2.0 styled. And this addon is off by default when you first install the extension. Can you elaborate?
I'd be ok to allow specific parts of the 2.0 to not be styled by this theme, feel free to open an issue
